runtime/browser/runtime_url_request_context_getter.cc

```diff
@@ -139,7 +159,7 @@ net::URLRequestContext* RuntimeURLRequestContextGetter::GetURLRequestContext() {
             net::DISK_CACHE,
             net::CACHE_BACKEND_DEFAULT,
             cache_path,
-            0,
+            GetDiskCacheSize(),
```

Comment:
The URLRequestContextGetter should only be called once per ContentBrowserClient object. Is it necessary to cache it?